File: Ejercicio_03/Domain/Entities/Departamento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Departamento
    {
        #region atributos
        private int _id;
        private string _nombre;
        #endregion

        #region constructores
        public Departamento() { }
        public Departamento(int id, string nombre)
        {
            _id = id;
            _nombre = nombre;
        }
        #endregion

        #region propiedades
        public int id 
        {
            get { return _id; }
        }
        public string nombre 
        {
            get { return _nombre; }
            set { _nombre = value; }
        }
        #endregion
    }
}

File: Angular/CRUD/src/domain/entities/Departamento.ts
export class Departamento {
    
    private _id: number
    private _nombre: string
    
    constructor(id: number, nombre: string) {
        this._id = id
        this._nombre = nombre
    }

    public get id(): number {
        return this._id
    }

    public set id(id: number) {
        this._id = id
    }

    public get nombre(): string {
        return this._nombre
    }

    public set nombre(nombre: string) {
        this._nombre = nombre
    }
}